test/oli/delivery/metrics/learning_proficiency_test.exs

```diff
@@ -397,25 +404,25 @@ defmodule Oli.Delivery.Metrics.LearningProficiencyTest do
       module_1: module_1,
       module_2: module_2
     } do
-      set_snapshot(section, page_1.resource, page_1_obj.resource, student_1, true)
-      set_snapshot(section, page_1.resource, page_1_obj.resource, student_2, true)
-      set_snapshot(section, page_1.resource, page_1_obj.resource, student_3, true)
-      set_snapshot(section, page_1.resource, page_1_obj.resource, student_4, true)
-
-      set_snapshot(section, page_2.resource, page_2_obj.resource, student_1, true)
-      set_snapshot(section, page_2.resource, page_2_obj.resource, student_2, false)
-      set_snapshot(section, page_2.resource, page_2_obj.resource, student_3, false)
-      set_snapshot(section, page_2.resource, page_2_obj.resource, student_4, false)
-
-      set_snapshot(section, page_3.resource, page_3_obj.resource, student_1, false)
-      set_snapshot(section, page_3.resource, page_3_obj.resource, student_2, true)
-      set_snapshot(section, page_3.resource, page_3_obj.resource, student_3, false)
-      set_snapshot(section, page_3.resource, page_3_obj.resource, student_4, false)
-
-      set_snapshot(section, page_4.resource, page_4_obj.resource, student_1, false)
-      set_snapshot(section, page_4.resource, page_4_obj.resource, student_2, true)
-      set_snapshot(section, page_4.resource, page_4_obj.resource, student_3, false)
-      set_snapshot(section, page_4.resource, page_4_obj.resource, student_4, false)
+      set_snapshots(section, page_1.resource, page_1_obj.resource, student_1, true)
```

Comment:
In order to show any proficiency for v1, we might have at least 3 snapshot records so I changed this function being called to `set_snapshots` which will create 3 records of the same value to meet this new requirement